dev-util/spec-cleaner/spec-cleaner-1.0.6.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5} )
+EGIT_REPO_URI="https://github.com/openSUSE/spec-cleaner.git"
+inherit distutils-r1
+[[ ${PV} == 9999 ]] && inherit git-r3
+
+DESCRIPTION="SUSE spec file cleaner and formatter"
+HOMEPAGE="https://github.com/openSUSE/spec-cleaner"
+[[ ${PV} != 9999 ]] && SRC_URI="https://github.com/openSUSE/${PN}/archive/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+[[ ${PV} != 9999 ]] && \
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="
+	test? (
+		dev-python/pytest
+		dev-python/pytest-cov
+		dev-python/pytest-runner
```

Comment:
You definitely don't need pytest-cov nor pytest-runner…

Reply:
Are you sure? https://github.com/openSUSE/spec-cleaner/blob/master/tox.ini#L14